lang/el.json

```diff
@@ -108,17 +108,17 @@
 	"ErrorUnknown": "Η ενέργεια δεν ήταν δυνατόν να εκτελεστεί. (Σφάλμα %1)",
 	"Errors": {
 		"10": "Λανθασμένη Εντολή.",
-		"11": "Το resource type δεν ήταν δυνατόν να προσδιοριστεί.",
-		"12": "Το resource type δεν είναι έγκυρο.",
-		"13": "The connector configuration file is invalid.",
-		"14": "Invalid connector plugin: %1.",
+		"11": "Ο τύπος πόρου(resource type) δεν ήταν δυνατόν να προσδιοριστεί.",
+		"12": "Ο τύπος πόρου(resource type) δεν είναι έγκυρος.",
+		"13": "Το αρχείο ρυθμίσεων του πρόσθετου(plugin) σύνδεσης είναι άκυρο.",
```

Comment:
To make things more clear here and below: the "connector" is simply the server-side part of CKFinder, it's not an additional plugin.